### DISCUSSION TITLE
things that can improve psych engine

### DISCUSSION THREAD

**Discussion by juliouveg** [FunkinCrew/Funkin#3141] [Ideas]
Hello, I want to report something that could improve Psych Engine 1.0.0. I would like someone not to crash when playing heavy mods and to read some mods because some scripts no longer work and if they changed something, let me know why the command was changed so that I can update all the mods. to this version

**Comment by AbnormalPoof** [comment]
![YEP](https://github.com/user-attachments/assets/4cd6aac3-a038-4676-86fe-11d4be4794ca)

**Reply by AbnormalPoof** [reply]
(This isn't Psych Engine, this is the main Funkin' repo.)

**Comment by moxie-coder** [comment]
> Hello, I want to report something that could improve Psych Engine 1.0.0. I would like someone not to crash when playing heavy mods and to read some mods because some scripts no longer work and if they changed something, let me know why the command was changed so that I can update all the mods. to this version

this isn’t the psych engine repo

**Comment by EVANEEEEEE** [comment]
KILL YOURSELF

**Reply by mayo78** [reply]
thats incredibly aggressive for no reason was this really needed

**Comment by charlesisfeline** [comment]
> Hello, I want to report something that could improve Psych Engine 1.0.0. I would like someone not to crash when playing heavy mods and to read some mods because some scripts no longer work and if they changed something, let me know why the command was changed so that I can update all the mods. to this version

does this place look `ShadowMario/FNF-PsychEngine` to you??

**Comment by SMB-5** [comment]
why is this here?

**Comment by Imatriceratops** [comment]
Im crying rn bro @juliouveg THIS IS THE WRONG REPO ARE YOU STUPID

**Comment by DarkWeBareBears69** [comment]
> Hello, I want to report something that could improve Psych Engine 1.0.0. I would like someone not to crash when playing heavy mods and to read some mods because some scripts no longer work and if they changed something, let me know why the command was changed so that I can update all the mods. to this version

Dude, this is the original Friday Night Funkin' repo, go to the official Psych Engine Discord server.

**Comment by JVNpixels** [comment]
Please close the discussion, ignore the death threats, you matter, but please close this discussion!
